spec/models/miq_report_result/purging_spec.rb

```diff
@@ -111,15 +111,18 @@
       end
     end
 
+    # private method - not sure if we need
     it "#purge_ids_for_remaining" do
       described_class.send(:purge_ids_for_remaining, 1).should == {1 => @rr1.last.id, 2 => @rr2.last.id}
     end
 
+    # private method - not sure if we need (and this is very expensive)
     it "#purge_counts_for_remaining" do
```

Comment:
This is used by the tools/purge_metrics.rb script (and other purge scripts as well).  The purposes of those scripts it to allow the customer to run offline purging.  The default "mode" for the script is to count, to allow it to act like a dry-run.